tests/adapter/exists.ts

```diff
@@ -0,0 +1,37 @@
+import { expect } from "chai";
+import nock from "nock";
+
+import ImageKitAdapter from "../../src";
+import { mockImage, mockNonExistentImage, testConfig } from "../fixtures";
+
+const imagekitAdapter = new ImageKitAdapter(testConfig());
+
+describe("exists", function () {
+  it("Should return true when image exists", async function () {
+    nock(`https://ik.imagekit.io`)
+      .get(`/test/${mockImage.targetDir}/${mockImage.name}`)
+      .reply(200);
+
+    const exists = await imagekitAdapter.exists(
+      mockImage.name,
+      mockImage.targetDir
+    );
+
+    expect(exists).to.equals(true);
+  });
+
+  it("Should return false when image does not exist", async function () {
```

Comment:
Let's add one more for private file.

Reply:
Done. Added for signed URL setting also.